content/en/01-about-pixie/06-troubleshooting.md

```diff
@@ -113,7 +113,9 @@ Pixie's [continuous profiler](/tutorials/pixie-101/profiler/) currently supports
 
 This is expected behavior. Pixie stores the data it collects in-memory on the nodes in your cluster; data is not sent to any centralized backend cloud outside of the cluster. So what you are observing is simply the data that it is collecting.
```

Comment:
Let's add an FAQ about OOMKilling.

<paraphrased>
Question: why is pixie oomkilling?
Answer:  this happens when Pixie is using more memory than the limit. Try increasing the limit, or reducing the amount of data stored in the table store. The amount of memory needed for Pixie depends on the application, but a minimum of 1Gi is required.

Reply:
I'm going to add this to the Troubleshooting page (not FAQ).